src/amo/components/RatingManagerNotice/styles.scss

```diff
@@ -1,10 +1,10 @@
 @import '~amo/css/styles';
 
-.RatingManagerNotice-savedRating {
-  margin-bottom: 12px;
-  text-align: center;
+.RatingManager .AddonReviewCard-allControls {
+  padding-bottom: 3px;
+  padding-top: 2px;
```

Comment:
This feels like a fragile way to keep the two blocks in sync.

I think the key to fixing this is to make the `light` notices appear like any other line of text. Here are a few ideas. What do you think?

```diff
diff --git a/src/amo/components/RatingManagerNotice/styles.scss b/src/amo/components/RatingManagerNotice/styles.scss
index 69990f701..5b723d982 100644
--- a/src/amo/components/RatingManagerNotice/styles.scss
+++ b/src/amo/components/RatingManagerNotice/styles.scss
@@ -1,10 +1,5 @@
 @import '~amo/css/styles';
 
-.RatingManager .AddonReviewCard-allControls {
-  padding-bottom: 3px;
-  padding-top: 2px;
-}
-
 .RatingManager .RatingManagerNotice-savedRating-hidden {
   display: none;
 }
diff --git a/src/ui/components/Notice/styles.scss b/src/ui/components/Notice/styles.scss
index 30fb9a1e3..5bdb9624a 100644
--- a/src/ui/components/Notice/styles.scss
+++ b/src/ui/components/Notice/styles.scss
@@ -14,6 +14,7 @@
     background-color: transparent;
     display: flex;
     justify-content: center;
+    line-height: $default-line-height;
     padding: 0;
   }
 
@@ -45,6 +46,14 @@
   width: 16px;
 }
 
+.Notice-light .Notice-icon {
+  // Reduce the icon size for light notices so that the total
+  // height of the notice adds up to $default-line-height.
+  background-size: 11px 11px;
+  height: 11px;
+  width: 11px;
+}
+
 .Notice-dismisser-button {
   cursor: pointer;
   padding: 0;
@@ -55,6 +64,10 @@
   margin: 0;
   padding: 4px 0;
 
+  .Notice-light & {
+    padding: 0;
+  }
+
   // The spec calls for additional spacing
   // in front of the action button. This is
   // applied even without a button to strike
```

Reply:
Nice! That works well. I'll update the css. Thanks @kumar303.